config/default/testRunner.conf.php

```diff
@@ -388,6 +388,10 @@
         ],
         'previous' => [
             'trigger' => 'K'
+        ],
+        'dialog' => [
+            'accept' => 'Enter',
+            'reject' => 'X'
```

Comment:
The reason why is the ESC key has an inconsistent behaviour, from the user's point of view.

For instance, when the page is in fullscreen mode, the ESC key is bound to the browser at a low level, to always allow the user to be able to leave this mode. And while the browser is in fullscreen mode, this key is unreachable from JavaScript. So if the user want to cancel the dialog, he/she will first escape the fullscreen, then will have to press ESC again to actually cancel the dialog.